gimie/parsers/cff.py

```diff
@@ -134,20 +134,25 @@ def get_cff_doi(data: bytes) -> Optional[str]:
 
     Examples
     --------
-    >>> get_cff_doi(bytes("doi:   10.5281/zenodo.1234", encoding="utf8"))
+    >>> get_cff_doi(bytes("identifiers:\\n    - type: doi\\n      value: 10.5281/zenodo.1234", encoding="utf8"))
     'https://doi.org/10.5281/zenodo.1234'
     >>> get_cff_doi(bytes("abc: def", encoding="utf8"))
-
     """
 
     try:
         cff = yaml.safe_load(data.decode())
     except yaml.scanner.ScannerError:
         logger.warning("cannot read CITATION.cff, skipped.")
         return None
-
     try:
-        doi_url = doi_to_url(cff["doi"])
+        identifiers = cff.get("identifiers", [])
+        doi_identifier = next(
```

Comment:
**question:** iiuc we arbitrarily, in case there are multiple identifiers, we aribitrarily pick the first one. The opposite would be to return a collection of all DOIs available. Is this desirable?

Reply:
**comment:** If we keep above behaviour, it may be good to mention it in the docstring.

Reply:
Ouuui that's a good point. I think the collection is more desirable than a random pick. Even if that means I will have to painfully refactor imaging plaza to allow for multiple DOI's, it is possible that a single piece of software has multiple publications. To pick the first one I think is too arbitrary.